src/registry.rs

```diff
@@ -102,7 +104,11 @@ impl Registry {
 
         for (index, worker) in workers.into_iter().enumerate() {
             let registry = registry.clone();
-            thread::spawn(move || unsafe { main_loop(worker, registry, index) });
+            let mut b = thread::Builder::new();
+            if let Some(name) = configuration.base_thread_name(index) {
+                b = b.name(name);
+            }
+            b.spawn(move || unsafe { main_loop(worker, registry, index) });
```

Comment:
Oh, it actually did in travis:

    warning: unused result which must be used, #[warn(unused_must_use)] on by default

Now I'm surprised this isn't default-deny, but perhaps we should set that ourselves.